doc/develop/dataio.rst

```diff
@@ -104,6 +105,48 @@ To read data from an OME-TIFF file, one could write:
    print(das[0].metadata[model.MD_LENS_MAG])
    print(das[0].metadata)
 
+Image position metadata
+=======================
+Most of the data store correspond to "spatial" data, i.e., an image representing
+the with axes in X and Y. In order to ensure a perfect overlay between data from
+different acquisition type, the metadata describes precisely how the image should
+be positioned. The following metadata has influence:
+
+* ``MD_PIXEL_SIZE``: size of a pixel (in m) in X and Y. In other words, this is the scale.
+* ``MD_POS``: the position (in m) of the *center of the image* in X and Y
+* ``MD_ROTATION``: counter-clockwise rotation (in radians) applied to the image from its center
+* ``MD_SHEAR``: *vertical* shear
+
+All values default to 0, excepted for the ``MD_PIXEL_SIZE`` which is required.
+
+Note that during acquisition, all these metadata have a twin-brother named with an extra
+`_COR` in order to record the correction on the image display. The function ``util.img.mergeMetadata()``
+can be used to merge these corrections into the main metadata. Before saving data
+to a file, the correction is automatically merged. So typically, after opening a
+file, the data will not have any of these extra correction metadata.
+
+For a given pixel situated at coordinates *p = i, j* in an image, its "physical" position
+*P = x, y* (in meters) can be computed with the following formula *P = RSLp + T*,
+where R is the rotation, L is the scale, and S is the shear.
```

Comment:
According to the docstring and code of the  `AffineTransform` class S is scale and L is shear. However I think the order should be `P = Rotation*Shear*Scale*p+Translation`. There might be a bug in the code.

Reply:
Well, the code is `numpy.dot(numpy.dot(R, S), L)`, so it's "RSL". Why do you think this would be wrong? It's all about convention anyway. In practice, we've tried a lot of images on the display, and it seems to always match perfectly, using this formula.

Reply:
It is indeed all about convention, and if you've tried with a lot of images it should be correct.

My comment that S is scale and L is shear in `AffineTransform`, is still true. So please update your explanation.

Reply:
Oooohhh... of course. Sorry, I didn't realise.